src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -226,7 +226,8 @@
   public static final String DB_TIMEZONE_DEFAULT = "UTC";
   private static final String DB_TIMEZONE_CONFIG_DOC =
       "Name of the JDBC timezone that should be used in the connector when "
-          + "querying with time-based criteria. Defaults to UTC.";
+          + "querying with time-based criteria.  Valid timezones are those given by "
+          + "TimeZone.getAvailableIDs().Defaults to UTC.";
```

Comment:
Is there value in having this code in the docs, or can we just say any valid JDK time zones are allowed?

Reply:
I think you're right. My original idea was to specify since JDK 7 and JDK 8 have slightly different support, but your suggestion of "any valid JDK time zone" leaves it to the user to use the correct input for their own JDK version